encoder/vaapiencoder_h264.cpp

```diff
@@ -1319,9 +1321,10 @@ bool VaapiEncoderH264::addSliceHeaders (const PicturePtr& picture) const
         sliceParam->slice_qp_delta = initQP() - minQP();
         if (sliceParam->slice_qp_delta > 4)
             sliceParam->slice_qp_delta = 4;
-        sliceParam->slice_alpha_c0_offset_div2 = 2;
-        sliceParam->slice_beta_offset_div2 = 2;
 
+        sliceParam->disable_deblocking_filter_idc = !m_videoParamAVC.enableDeblockFilter;
+        sliceParam->slice_alpha_c0_offset_div2 = m_videoParamAVC.deblockAlphaOffset;
```

Comment:
Hi @lizhong1008 & @wudping 
two question:
1. original sliceParam->slice_alpha_c0_offset_div2 default is 2, this code will change it to 0. Do you think this is ok? 
2. slice_alpha_c0_offset_div2 means offset div 2, but we can't get this hint from deblockAlphaOffset' name, Is it OK?